frontend/src/metabase/reference/utils.unit.spec.js

```diff
@@ -124,150 +148,142 @@ describe("Reference utils.js", () => {
 
     it("should generate correct question for table raw data", () => {
       const question = getQuestion({
-        dbId: 3,
-        tableId: 4,
+        dbId,
+        tableId,
+        metadata,
       });
 
-      expect(question).toEqual(
-        getNewQuestion({
-          database: 3,
-          table: 4,
-        }),
-      );
+      expect(question).toEqual(getNewQuestion());
     });
 
     it("should generate correct question for table counts", () => {
       const question = getQuestion({
-        dbId: 3,
-        tableId: 4,
+        dbId,
+        tableId,
         getCount: true,
+        metadata,
       });
 
       expect(question).toEqual(
         getNewQuestion({
-          database: 3,
-          table: 4,
-          aggregation: [["count"]],
+          aggregation: [["count", null]],
```

Comment:
I think the `null` should not be there. Looks like a backend issue. I asked [a question in Slack](https://metaboat.slack.com/archives/C04CYTEL9N2/p1705408133622419).

Reply:
[Reverted](https://github.com/metabase/metabase/pull/37694/commits/4509c48a34429f94317fd62224c379f966b780a9), [skipped the tests for now](https://github.com/metabase/metabase/pull/37694/commits/06fd8a707a1d39b21fb94e2f82e021c295bf8bbe) and filed a BE issue for it: #37782